contracts/crowdfund/Crowdfund.sol

```diff
@@ -366,24 +367,27 @@ abstract contract Crowdfund is Implementation, ERC721Receiver, CrowdfundNFT {
     /// @param initialDelegates The addresses to delegate to for each recipient.
     /// @param values The ETH to contribute for each recipient.
     /// @param gateDatas Data to pass to the gatekeeper to prove eligibility.
-    /// @param revertOnFailure If true, revert if any contribution fails.
     function batchContributeFor(
         address[] memory recipients,
         address[] memory initialDelegates,
-        uint256[] memory values,
-        bytes[] memory gateDatas,
-        bool revertOnFailure
```

Comment:
I don't think it does, but note to self that we'll have to check with FE this doesn't break anything.

Reply:
good callout. Agree that it's probably fine.